aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -232,15 +289,15 @@ impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> VersionedGro
         Ok(())
     }
 
-    fn get_committed_group(&self) -> Vec<(T, Arc<V>)> {
+    fn get_committed_group(&self) -> Vec<(T, ValueWithLayout<V>)> {
         self.committed_group.clone().into_iter().collect()
     }
 
     fn get_latest_tagged_value(
-        &self,
+        &mut self,
```

Comment:
why mut? I don't see a change below that requires mut?

Reply:
Artifact of previous design. Removed it.